Framework/Core/include/Framework/Variant.h

```diff
@@ -18,6 +18,7 @@
 #include <iosfwd>
 #include <initializer_list>
 #include <string_view>
+#include <vector>
```

Comment:
Is this really needed?

Reply:
`build/O2/o2` was failing by seemingly not recognizing `vector`, so this is a workaround